lib/std/os.zig

```diff
@@ -1185,7 +1185,7 @@ pub fn getenvW(key: [*:0]const u16) ?[:0]const u16 {
         while (ptr[i] != 0) : (i += 1) {}
         const this_value = ptr[value_start..i :0];
 
-        if (mem.eql(u16, key_slice, this_key)) return this_value;
+        if (std.ascii.eqlIgnoreCase(mem.sliceAsBytes(key_slice), mem.sliceAsBytes(this_key))) return this_value;
```

Comment:
I'm pretty sure this will erroneously compare the high byte(s) in actual unicode as ascii.

Reply:
Yup, good catch. Not sure what the best solution would be here :(

Reply:
Yeah, I think so too. We could either add an ascii check to `std.ascii.eqlIgnoreCase` or a new private function here that makes the check.